dashboard/app/models/user.rb

```diff
@@ -861,7 +884,10 @@ def self.find_for_authentication(tainted_conditions)
       ).first
     elsif hashed_email = conditions.delete(:hashed_email)
       return nil if hashed_email.utf8mb4?
-      where(hashed_email: hashed_email).first
+      legacy = where(hashed_email: hashed_email).first
+      matching_auth_opts = AuthenticationOption.where(credential_type: 'email', authentication_id: hashed_email)
```

Comment:
Our tech spec examples show `"hashed_email"` as the credential type, but it looks like you're using `"email"`.  I'm fine with the shorter string, the meaning's pretty unambiguous.  Want me to update the tech spec?

Reply:
Hmm, very good point...I pulled `email` from earlier tests, and I think it makes sense. I'll update the spec.